binding.js

```diff
@@ -95,19 +95,22 @@ module.exports.download_vips = function () {
           } catch (err) {}
         });
       });
-      const gotOpt = {
+      const url = distBaseUrl + tarFilename;
+      const simpleGetOpt = {
+        url,
```

Comment:
This will probably need to be `url: url,` as Node v4 lacks destructuring support.

Reply:
will do